src/ui/buttons/button-animations/index.js

```diff
@@ -9,7 +9,7 @@ import { type ButtonAnimationOutputParams } from './types';
 
 export function getButtonAnimation(personalization : ?Personalization) : ButtonAnimationOutputParams | Object {
     // Only show animations for SSR
-    if (__WEB__ || !personalization) {
+    if (__WEB__ || !personalization || !personalization.buttonAnimation) {
```

Comment:
What do you think about moving this validation to a variable? Here's what I'm thinking:
```js
const isValidPersonalization = personalization && personalization === 'object' && personalization.buttonAnimation;
if (__WEB__ || !isValidPersonalization)  {   
    return {};
}
```